code/modules/events/blob.dm

```diff
@@ -6,6 +6,8 @@
 
 	earliest_start = 48000 // 1 hour 20 minutes
 
+	gamemode_blacklist = list("blob") // Just in case blob survives that long

```

Comment:
Fun fact the blob mode manually disables the event by setting how many times it can happen to 0